### PR TITLE
[#151, #152] Enables SSL and updates feature branch before deliver

### PR DESCRIPTION
**Pull request by codenamev**
[#151, #152] Enables SSL and updates feature branch before deliver

### CODE REVIEW

File: lib/git_reflow/git_helpers.rb [hunk 1]
```diff
@@ -31,6 +31,11 @@ def push_current_branch
       run_command_with_label "git push origin #{current_branch}"
     end
 
+    def update_current_branch
+      run_command_with_label "git pull origin #{current_branch}"
```
Comment (nhance) [inline lib/git_reflow/git_helpers.rb]: What happens if we have merge conflicts?

Comment (codenamev) [inline lib/git_reflow/git_helpers.rb]: just like any command it will error and stop any further commands

**Comment by nhance** [comment]
lgtm

File: lib/git_reflow/git_helpers.rb (hunk 1 above)
Comment (pboling) [inline lib/git_reflow/git_helpers.rb]: 👍

**Pull request closed without merging**